drops/azure_edge_extensions_rag_on_edge_multimodal.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Title": "Multi-Modal Retrieval-Augmented Generation on Edge with Azure Arc & Azure IoT Operations",
+  "Summary": "This Drop provides a multi-modal (image + text) Retrieval-Augmented Generation(RAG) on edge pattern that facilitates facilitates multi-modal text and image searches within files utilizing a multi-modal vector search engine Marqo and an multi-modal Large Language Model LLaVa.",
+  "Description": "Multimodal RAG on Edge is a tool to perform multi-modal searches, which allows image, text, or combination of both as the input query, within files using a multi-modal vector search engine Marqo, and to generate a readable response based on the search result with LLAVA.
+The tool can help on machine troubleshooting on-site and more use cases. In this sample, we demonstrate how to use the tool as industry copilot to help people on construction site quickly troubleshoot the excavators. During the construction operation, When the excavators is broken, it is usually time-consuming for the staff on site to find the solution from the technical manual and the past troubleshooting logs or contact technical specialist for help. The edge multi-modal RAG tool serves as the industry copilot helping the operator staff to quickly fix the problems.
+The solution is independent from cloud services, and can be deployed to the edge device.",
+  "Cover": "",

```

Comment:
I see that you have a readlly good video explaining the demo -> https://microsoftapc-my.sharepoint.com/personal/chencheng_microsoft_com/_layouts/15/stream.aspx?id=%2Fpersonal%2Fchencheng%5Fmicrosoft%5Fcom%2FDocuments%2FVideos%2FClipchamp%2FVideo%20Project%204%2FExports%2FMulti%2DModal%2DRAG%2Don%2DEdge%2Emp4&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0&ga=1&referrer=StreamWebApp%2EWeb&referrerScenario=AddressBarCopied%2Eview%2Ebdd0ad65%2Df461%2D4305%2D9344%2D1e4e243ccf0e


My suggestion is to publish this video publicly and add the link to the "Cover" to be visible for users

Reply:
updated